### PR TITLE
Task/APPS-1641 — About page

### PR DESCRIPTION
**Pull request by giuliacampana**
Still waiting on people's photos.

Screen Shot 2020-01-14 at 12 41 31 PM
Screen Shot 2020-01-14 at 12 41 39 PM
Screen Shot 2020-01-14 at 12 42 21 PM
Screen Shot 2020-01-14 at 12 42 29 PM

### CODE REVIEW

File: src/containers/about/InvestorsSection.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+const Header = styled(SectionHeader)`
+  margin-bottom: 0;
+`;
+
+const LogosOuterWrapper = styled.div`
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: row;
+  justify-content: center;
```
Comment (anzioka) [inline src/containers/about/InvestorsSection.js]: `flex-direction: row` is default.

File: src/containers/about/InvestorsSection.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import { MEDIA_QUERY_LG, MEDIA_QUERY_MD } from '../../core/style/Sizes';
+
+const Content = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 100px auto 100px auto;
+  text-align: center;
+  width: 100%;
```
Comment (anzioka) [inline src/containers/about/InvestorsSection.js]: `margin: 100px auto` is more compact, I think.

File: src/containers/about/MeetTheTeam.js [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+import { MEDIA_QUERY_LG, MEDIA_QUERY_MD, MEDIA_QUERY_TECH_SM } from '../../core/style/Sizes';
+import { TEAM_NAMES, TEAM_PHOTOS, TEAM_ROLES } from './TeamMemberConsts';
+
+const Content = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto 0 auto;
+  text-align: center;
```
Comment (anzioka) [inline src/containers/about/MeetTheTeam.js]: `margin: 0 auto`

File: src/containers/about/MeetTheTeam.js [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+  position: absolute;
+  text-align: center;
+  top: 360px;
+  width: 100%;
+`;
+
+const TeamMembersOuterWrapper = styled.div`
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: row;
```
Comment (anzioka) [inline src/containers/about/MeetTheTeam.js]: `flex-direction: row` is default

File: src/containers/about/MeetTheTeam.js [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.ANDREA} />
+            <TeamMemberName>{TEAM_NAMES.ANDREA}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.ANDREA}</TeamMemberRole>
+          </TeamMember>
+          <TeamMember>
+            <TeamMemberPhoto headshot={TEAM_PHOTOS.MANDY} />
+            <TeamMemberName>{TEAM_NAMES.MANDY}</TeamMemberName>
+            <TeamMemberRole>{TEAM_ROLES.MANDY}</TeamMemberRole>
+          </TeamMember>
```
Comment (anzioka) [inline src/containers/about/MeetTheTeam.js]: I think the map () function should be used here.

File: src/containers/products/ProductsList.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  font-size: 18px;
+  margin-bottom: 80px;
+  width: 524px;
+`;
+
+const ProductTilesOuterWrapper = styled.div`
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: row;
+  justify-content: center;
```
Comment (anzioka) [inline src/containers/products/ProductsList.js]: `flex-direction: row` is default

File: src/containers/about/MeetTheTeam.js (hunk 5 above)
Comment (giuliacampana) [reply src/containers/about/MeetTheTeam.js]: Same deal as: https://github.com/Lattice-Works/openlattice.com/pull/202#discussion_r366556191.

**Pull request merged**